src/Panel/DeprecationsPanel.php

```diff
@@ -93,20 +93,9 @@ protected function _prepare()
             }
         }
 
-        /** @psalm-suppress RedundantFunctionCall */
-        ksort($return['app']);
-        /** @psalm-suppress RedundantFunctionCall */
-        ksort($return['cake']);
-        /** @psalm-suppress RedundantFunctionCall */
         ksort($return['plugins']);
-        /** @psalm-suppress RedundantFunctionCall */
         ksort($return['vendor']);
```

Comment:
Why are some sorts removed?

Reply:
@markstory These array were lists, not associative arrays, so using key based sorting was redundant.